base/int.jl

```diff
@@ -174,8 +174,15 @@ julia> abs(-3)
 julia> abs(1 + im)
 1.4142135623730951
 
-julia> abs(typemin(Int64))
--9223372036854775808
+julia> abs.(typemin(Int8):Int8(-125))  # overflow at typemin
```

Comment:
Can this use less vertical space? The existing example that was deleted already shows the point quite succinctly in my opinion.

Reply:
I pushed a more compact one,`abs.(Int8[-128 -127 -126 0 126 127])`. 

What it wants to illustrate is that this is an effect at exactly one number.